src/main/java/unbanner/StudentController.java

```diff
@@ -11,6 +11,7 @@
     @Autowired
     StudentRepository repository;
 
+    //list all students
```

Comment:
More documentation.

What does 'all' mean? Where do 'all students' come from? What is the listing format?